x/ibc/04-channel/client/cli/tx.go

```diff
@@ -288,6 +300,223 @@ func GetMsgChannelCloseConfirmCmd(storeKey string, cdc *codec.Codec) *cobra.Comm
 	}
 }
 
+func GetCmdHandshake(storeKey string, cdc *codec.Codec) *cobra.Command {
```

Comment:
I thought we wanted to split the handshake into its individual commands?  cc: @cwgoes

Reply:
We do, we should do that.